sdk/core/System.ClientModel/src/System.ClientModel.csproj

```diff
@@ -6,7 +6,7 @@
     <Nullable>enable</Nullable>
     <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
     <NoWarn>$(NoWarn);AZC0001</NoWarn>
-    <PackageIcon>DotNetPackageIcon.png</PackageIcon>
+    <PackageIcon>DotNetPackageIconNew.png</PackageIcon>
```

Comment:
We recently removed System.Memory.Data which was the only project that used the other icon so I think it is safe to just overwrite the existing icon instead of having this side-by-side.